### PR TITLE
Artik #108 Cannot open the terminal of ARTIK board, after connect multiple boards

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
Fixes position of buttons in consoles panel

### CODE REVIEW

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/panel/ProcessesPanelPresenter.java [hunk 1]
```diff
@@ -711,6 +711,14 @@ private void restoreState(final Machine machine) {
             @Override
             public void apply(List<MachineProcessDto> arg) throws OperationException {
                 for (MachineProcessDto machineProcessDto : arg) {
+                    /**
+                     * Do not show the process if the command line has prefix #hidden
+                     */
+                    if (machineProcessDto.getCommandLine() != null && !machineProcessDto.getCommandLine().isEmpty()
```
Comment (vparfonov) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/panel/ProcessesPanelPresenter.java]: Please, use here Strings.isNullOrEmpty() (from Guava library)

Comment (vitaliy-guliy) [reply plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/panel/ProcessesPanelPresenter.java]: Done

**Comment by vparfonov** [comment]
thank you very much

**Comment by vparfonov** [comment]
ok

**Pull request merged**